jest.config.js

```diff
@@ -3,4 +3,8 @@ module.exports = {
   collectCoverageFrom: ["src/**/*.{js,jsx,ts,tsx}", "!**/node_modules/**"],
   coverageDirectory: "coverage",
   testEnvironment: "jsdom",
+  preset: 'ts-jest',
```

Comment:
it's already added by @hubert-deriv , please remove this or sync your branch with master.